xarray/structure/alignment.py

```diff
@@ -837,7 +837,7 @@ def align(
     >>> a, b = xr.align(x, y, join="exact")
     Traceback (most recent call last):
     ...
-    AlignmentError: cannot align objects with join='exact' ...
+    xarray.structure.alignment.AlignmentError: cannot align objects with join='exact' ...
```

Comment:
Will be cleaner to later have Xarray exceptions defined in their own module `xarray.exceptions.AlignmentError` (same for `MergeError`, etc.)